.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<h4> Summary (Short summary of what is being done) : </h4>
```

Comment:
Can we add descriptions ("(Short summary of what is being done)") as alt text?

Reply:
Not sure what you mean by Alt text. AFAIK Alt texts are for image captions when image is not loaded. Do you mean italics? Either ways, I'd personally prefer the template to be simple enough as this is just a guideline for the PR authors to make sure the PR carries the relevant context.

Reply:
Will the user remove the description when generating the PR then?

Reply:
Writing it in italics can help. We can overwrite it when generating PRs.

Reply:
The description is just to help user understand what is being wanted in the section. IMHO User can just fill the template after the field ends.